### ISSUE TITLE
ats-metadata

### ISSUE THREAD

**Issue by aquaglia** [inspire-eu-validation/ats-metadata#3]
I question the following statement:
"gmx:Anchor is typically used to reference a URI on which the characterstring content is available. The validator should resolve the URI in the gmx:Anchor to validate if the content is available."

I think that gmx:Anchor.href is not intended to resolve to a plain characterstring, rather it 
"supports hyper-linking capabilities and ensures a web-like implementation of CharacterStrings"
In other words, a full-blown web page containing also images, that describes the plain text included in the gmx:Anchor element value.

**Comment by fxprunayre** [comment]
We discussed that with @pvgenuchten and examples should be added to the wiki page for gmx:Anchor and multilingual records (with only PT_FreeText or PT_FreeText+CharacterString). And you're right, when having and Anchor, there is no need to resolve the link (in href attrbute) but use the value of the Anchor.

Example of an INSPIRE keyword which should be considered valid (from my point of view):

```
  <gmd:keyword>
    <gco:CharacterString>Addresses</gco:CharacterString>
  </gmd:keyword>
or
  <gmd:keyword>
    <gmx:Anchor xlink:href="http://www.eionet.europa.eu/gemet/en/inspire-theme/5297/">Addresses</gmx:Anchor>
  </gmd:keyword>
or 
  <gmd:keyword>
    <gmd:PT_FreeText>
       <gmd:textGroup>
         <gmd:LocalisedCharacterString locale="#EN">Addresses</gmd:LocalisedCharacterString>
       </gmd:textGroup>
    </gmd:PT_FreeText>
  </gmd:keyword>
```

**Comment by aquaglia** [comment]
Yes, I agree now, thanks.
